### PR TITLE
11th Sep: Added limitation and url prefixes

### PR DESCRIPTION
**Pull request by Abhinandan578**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md [hunk 1]
```diff
@@ -32,30 +32,57 @@ The concept of Signed URLs is common across various cloud storage solutions for
...
 Please refer [here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-presigned-url.html) for more details.
 
-## Dropbox
+### Dropbox
 
 PDF Services APIs can connect with Dropbox using Signed URIs for downloading and uploading the files from Dropbox location without the need for any credentials or access token.
 
+#### Supported URL Prefixes:
+
+- `https://<uid of item>.dl.dropboxusercontent.com`
```
Comment (piyuverma) [inline src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md:66]: 'item-uid' or 'item-id' we can use ?

File: src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md [hunk 2]
```diff
@@ -32,30 +32,57 @@ The concept of Signed URLs is common across various cloud storage solutions for
...
 
 
-## SharePoint
+### SharePoint
 
 PDF Services APIs can connect with SharePoint using Signed URIs for downloading and uploading the files from SharePoint site. 
 
+#### Supported URL Prefixes
+
+- `https://<companyname>.sharepoint.com`
```
Comment (piyuverma) [inline src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md:41]: please write, 'company-name', as this is a placeholder.

**Review by piyuverma — CHANGES_REQUESTED** [review]

File: src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md (hunk 1 above)
Comment (Abhinandan578) [reply src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md]: done

File: src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md (hunk 2 above)
Comment (Abhinandan578) [reply src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md]: done. Changed to tenant-name

**Pull request merged**